book/typst/modules/appendix2.typ

```diff
@@ -2,5 +2,347 @@
+constants):
+$   (1/(T - A)) (d T)/(d t) & = -k,\
+        ln(T - A) &= -k t + C , quad "note that " T - A > 0 ,\
+        T - A &= D e^(-k t),\
+        T &= A + D e^(-k t)
+$
+That is,
+$T = 22 + D e^(-k t)$. We plug in the first condition: $89 = T(0) = 22 +
+D$,
+and hence $D = 67$. So
+$T = 22 + 67 e^(-k t)$. The second condition says $85 = T(1) =
+22 + 67 e^(-k)$. Solving for $k$, we get
+$k = - ln ((85 - 22)/67) approx 0.0616$. Now we solve for the time $t$
+that gives us a temperature of 60 degrees. Namely, we solve
+$ 60 = 22 + 67 e^(-0.0616 t) $
+to get
+$t = - (ln ((60 - 22)/67))/(0.0616) approx 9.21$ minutes. So Bob can
+begin to drink the coffee at just over 9 minutes from the time Bob made
+it. That is probably about the amount of time it took us to calculate how long it would take.
+
+
+#{
+    align(center,
+        {
+            let x = lq.linspace(0, 10)
+            let y = x.map(x => 22 + 67 * calc.exp(-0.0616 * x))
+            let xticks = lq.linspace(0, 9, num: 10)
+            let yticks = lq.linspace(55, 85, num: 7)
+            lq.diagram(
+                xlim: (0, 10),
+                ylim: (55, 90),
+                // width: 5cm,
+                // height: 5cm,
+                yaxis: (position: 0, tip: tiptoe.stealth, ticks: yticks),
+                xaxis: (position: 55, tip: tiptoe.stealth, ticks: xticks),
+                lq.plot(
+                    x, 
+                    y,
+                    mark: none,
+                    stroke: (paint: blue)
+                ),
+                // lq.line(
+                //     (1, 55), 
+                //     (1, 90), 
+                //     stroke: (paint: gray, dash: "dashed")
+                // ),
+                lq.line(
+                    (9.21, 55), 
+                    (9.21, 90),
+                    stroke: (paint: gray, dash: "dashed")
+                    )
+            )
+        }
+    )
+}
+
+
+
+// #figure(
+//     image("coffeefig-1-2", width: 9in),
+//     caption: [
+//         Graphs of the coffee temperature function $T(t)$.
+//         On the left, horizontal
+//         lines are drawn at temperatures 60, 85, and 89. Vertical lines
+//         are drawn at $t = 1$ and $t = 9.21$. Notice that the
+//         temperature of the coffee hits 85 at $t = 1$, and 60 at
+//         $t approx 9.21$. On the right, the graph is over a longer period of time,
+//         with a horizontal line at the ambient temperature 22.
+//         (label: sintro:coffeefig)
+//     ]
+// )
+// ]
+]
+)<ex_coffee>
+
+#example(
+        prompt: [
+Find the general solution to $y' = (-x y^2)/3$
+(including any singular solutions).],
+[
+First note that $y = 0$ is a solution (a singular solution).
+Now assume that $y not = 0$.
+$ -3/y^2 y' &= x,\
+        3/y &= x^2/2 + C,\
+            y &= 3/((x^2)/2 + C)\
+              &= 6/(x^2 + 2C)
+$
+So the general solution is
+$ y = 6/(x^2 + 2C) quad "and" quad y = 0 . $
+]
+)
+
+== Exercises
```

Comment:
You should create a `*-practice-problems.typ` file and include the questions there like for the modules.

Reply:
But don't forget to `git add` the new file ;-)